Automattic/wc-calypso-bridge
# Fix JS lint errors

### Changes proposed in this Pull Request:



This PR fixes some existing JS lint errors (related to space, comma, indent..etc)

- Disable [react/react-in-jsx-scope](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md) rule since we're using React as a global variable.
- Fix `format:js` command (e82b994b6e32070eba91c4c26f77f351fba54e1a). See https://github.com/WordPress/gutenberg/tree/trunk/packages/scripts#format 
- Fix lint errors by running `npm run format:js`
- Manually fix some simple errors 91202323d3b54a5d808163ab59c0b98b93815b1c


There are still some lint errors but I like to resolve them when we work on those files again.






### How to test the changes in this Pull Request:

1. Review the changes
2. Run `nvm use`
3. Run `npm install`
4. Run `npm run format:js`  -> files should be unchanged.
5. Run `npm run build` should build files successfully
